From 30397a024f57f14800975bbb4312be54cc75202b Mon Sep 17 00:00:00 2001
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Tue, 6 Apr 2010 19:38:51 -0300
Subject: [PATCH] net: remove NICInfo.bootable field

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=561078

It is just set by net_set_boot_mask() and never used. The logic for rom loading
changed a lot since this field was introduced. It is not needed anymore.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 net.c |    1 -
 net.h |    1 -
 2 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/net.c b/net.c
index de7d626..5cebb1a 100644
--- a/net.c
+++ b/net.c
@@ -1204,7 +1204,6 @@ void net_set_boot_mask(int net_boot_mask)

     for (i = 0; i < nb_nics; i++) {
         if (net_boot_mask & (1 << i)) {
-            nd_table[i].bootable = 1;
             net_boot_mask &= ~(1 << i);
         }
     }
diff --git a/net.h b/net.h
index 33a1eaf..5b6e814 100644
--- a/net.h
+++ b/net.h
@@ -135,7 +135,6 @@ struct NICInfo {
     VLANState *vlan;
     VLANClientState *netdev;
     int used;
-    int bootable;
     int nvectors;
 };

-- 
1.6.6.1

